src/Aspire.Hosting/Publishing/PublishingActivityReporter.cs

```diff
@@ -263,12 +263,6 @@ private bool HasStepsInProgress()
 
     private async Task HandleInteractionUpdateAsync(Interaction interaction, CancellationToken cancellationToken)
     {
-        // Only handle input interaction types
-        if (interaction.InteractionInfo is not Interaction.InputsInteractionInfo inputsInfo || inputsInfo.Inputs.Count == 0)
```

Comment:
Do we have tests for this?

Reply:
PromptNotificationAsync_EmitsCorrectActivityAndHandlesCompletion covers handling for `PromptNotificationAsync`.